sdks/python/apache_beam/dataframe/expressions_test.py

```diff
@@ -53,6 +54,61 @@ def test_expression_proxy_error(self):
     with self.assertRaises(TypeError):
       expressions.ComputedExpression('add', lambda a, b: a + b, [a, b])
 
+  def test_output_partitioning_preserves_singleton(self):
+    a = expressions.PlaceholderExpression(1)
+    b = expressions.PlaceholderExpression(2)
+
+    preserves_singleton = expressions.ComputedExpression(
```

Comment:
Maybe call this `preserves_only_singleton`? And give an example (like setting the index from a column).

Reply:
Done